webapp/apps/dynamic/views.py

```diff
@@ -788,6 +784,26 @@ def behavior_results(request, pk):
                          'webapp_version': webapp_vers_disp}
 
     model = url.unique_inputs
+
+    first_year = model.first_year
```

Comment:
There are some cases from really old runs where `first_year` is `None`. This can be fixed with `first_year = model.first_year or START_YEAR`, defaulting to the current start year if one is not specified.

Reply:
If first_year is `None`, then that will break the `to_json_reform` call.